lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -146,6 +146,16 @@
                your playbook accordingly to not export the disk all the time.
                This option is valid only for template disks."
         version_added: "2.4"
+    host:
+        description:
+            - "When the hypervisor name is specified the newly created disk or
+               an existing disk will refresh its information about the
+               underlying storage( Disk size, Serial, Product ID, Vendor ID ...)
+               The specified host will be used for garherting the storage
```

Comment:
I think 'garherting' is not the spelling you want

Reply:
Roman, you probably wanted to use 'gathering', right?